### PR TITLE
Moves libs into dev section and adds clean command

### PR DESCRIPTION
**Pull request by RomanGalochkin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
+  "scripts": {
+    "lerna": "lerna",
+    "bootstrap": "lerna bootstrap",
+    "start": "node packages",
+    "dev": "nodemon packages",
+    "migrate": "yarn server/db:migrate",
+
+    "packages/build": "cd packages/guesstimator && yarn build && cd ../.. && cd packages/components && yarn build && cd ../.. && cd packages/prediction-analysis && yarn build && cd ../.. && cd packages/client && yarn build-t && cd ../..",
+    "packages/clean": "cd packages/guesstimator && yarn clean && cd ../.. && cd packages/components && yarn clean && cd ../.. && cd packages/prediction-analysis && yarn clean && cd ../.. && cd packages/client && yarn clean && cd ../..",
+    "packages/remove": "rm -rf ./packages/client/node_modules && rm -rf ./packages/components/node_module && rm -rf ./packages/prediction-analysis/node_module && rm -rf ./packages/guesstimator/node_module",
```
Comment (OAGr) [inline package.json]: why do these say ``/node_module`` instead of ``node_modules``?

**Comment by OAGr** [comment]
Kudos for getting this to build! One comment.

File: package.json (hunk 1 above)
Comment (RomanGalochkin) [reply package.json]: Sure, it is a mistake, will repair

Comment (OAGr) [reply package.json]: Sg, no worries, just checking

**Pull request merged**